internal/helper/helper.go

```diff
@@ -284,6 +284,68 @@ func SafeQuantitySubtraction(orig, subtract quantity.Size) quantity.Size {
 	return orig - subtract
 }
 
+// CreateTarArchive places all of the files from a source directory into a tar.
+// Currently supported are uncompressed tar archives and the following
+// compression types: zip, gzip, xz bzip2, zstd
+func CreateTarArchive(src, dest, compression string, verbose, debug bool) error {
+	tarFile, err := os.Create(dest)
+	if err != nil {
+		return fmt.Errorf("Error creating tar archive \"%s\": \"%s\"", dest, err.Error())
+	}
+	var fileWriter io.WriteCloser = tarFile
+	tarWriter := tar.NewWriter(fileWriter)
+	defer tarWriter.Close()
+	return filepath.Walk(src, func(filePath string, fileInfo os.FileInfo, err error) error {
```

Comment:
I'm surprised that go lintian and such is fine with such a huge lambda function! It's not that I don't like it, I think it's pretty nice.